python/download_dashboard_pdf.py

```diff
@@ -13,26 +15,30 @@ def main():
     """Given a dashboard title, search all dashboards to retrieve its id and use
     it to render the dashboard's pdf.
 
+    Examples of how to use this:
     $ python download_dashboard_pdf.py "A Test Dashboard"
+    $ python download_dashboard_pdf.py "A Test Dashboard" "{'filter1': 'value1, value2', 'filter2': 'value3'}"
+    $ python download_dashboard_pdf.py "A Test Dashboard" {} "single_column"
     """
     dashboard_title = sys.argv[1] if len(sys.argv) > 1 else ""
-    pdf_style = sys.argv[2] if len(sys.argv) > 2 else "tiled"
-    pdf_width = int(sys.argv[3]) if len(sys.argv) > 3 else 545
-    pdf_height = int(sys.argv[4]) if len(sys.argv) > 4 else 842
+    filters = ast.literal_eval(sys.argv[2]) if len(sys.argv) > 2 else None
```

Comment:
could we `json.loads` here instead? It just seems out of place to include the `ast` module in here :-)